cmd/ipfs-cluster-ctl/main.go

```diff
@@ -614,10 +614,23 @@ with "sync".
 
 When the --local flag is passed, it will only fetch the status from the
 contacted cluster peer. By default, status will be fetched from all peers.
+
+When the --filter is passed, it will only fetch the peer information
+where status of the pin matches with the filter value.
+Valid filter values are tracker status types("pinned", "pin_error", "unpinning" etc),
+an alias of tracker status type (queued or error), comma separated list of
+tracker status type and/or it aliases ("error,pinning")
+On passing invalid filter value no status information will be shown
+List of tracker status types
+https://github.com/ipfs/ipfs-cluster/blob/319c41cbf195b0453b8d1987991280d3121bac93/api/types.go#L66
```

Comment:
No point using the commit url, as we will have to update this url every time we update the list. May as well just put all the filter values here and add a comment in `api/types` to remind people that if they add to the list of filters, they need to update this comment with the new values.

Reply:
You can loop and pretty print that list here.  An invalid filter should throw an error (catching it both in the client (to avoid making a request for nothing) and in the server side (for people that doesn't use our client)).